app/views/docs/oauth-providers/spotify.md

```diff
@@ -140,7 +140,7 @@ import io.appwrite.services.Account
 
 val client = Client(context)
     .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
-    .setProject("[PROJECT_ID]") // Your project ID
+    .setProject("[PROJECT_ID]")                         // Your Project ID
```

Comment:
Did a quick review of all the comments, and they do not look aligned. Please align all of them.

Reply:
Aligning them all now; my bad 😅